client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -125,6 +125,19 @@ export const getTask = (
 				isSkippable: true,
 			};
 			break;
+		case CHECKLIST_KNOWN_TASKS.WOOCOMMERCE_SETUP:
+			taskData = {
+				timing: 2,
```

Comment:
@jeffgolenski Did you really mean 2 minutes to set up the store? I thought 7 was pretty short, but not totally unreasonable, but 2 seems a bit beyond optimistic. If this is the official line from the Woo team, I guess I can get on board, but...

Reply:
Yeah 2 minutes would be a bit of a rush. The Woo checklist has its own time estimates so we could base our estimate of theirs?
<img width="719" alt="Screenshot 2021-07-26 at 5 15 07 PM" src="https://user-images.githubusercontent.com/1500769/126936585-5c487c80-07d3-4a7f-b5aa-761ae52944ec.png">

So maybe "6+ minutes"?

Reply:
Just making a note to disregard that "2 minutes" indicator in my mock. It should remain at 7 if that's what it already was.